servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -2029,96 +455,96 @@ void RendererSceneRenderRD::_update_dirty_skys() {
 			}
 			texture_set_dirty = true;
 		}
 
 		// Create subpass buffers if they haven't been created already
-		if (sky->half_res_pass.is_null() && !RD::get_singleton()->texture_is_valid(sky->half_res_pass) && sky->screen_size.x >= 4 && sky->screen_size.y >= 4) {
+		if (skyrd->half_res_pass.is_null() && !RD::get_singleton()->texture_is_valid(skyrd->half_res_pass) && skyrd->screen_size.x >= 4 && skyrd->screen_size.y >= 4) {
 			RD::TextureFormat tformat;
 			tformat.format = RD::DATA_FORMAT_R16G16B16A16_SFLOAT;
-			tformat.width = sky->screen_size.x / 2;
-			tformat.height = sky->screen_size.y / 2;
+			tformat.width = skyrd->screen_size.x / 2;
+			tformat.height = skyrd->screen_size.y / 2;
 			tformat.usage_bits = RD::TEXTURE_USAGE_SAMPLING_BIT | RD::TEXTURE_USAGE_COLOR_ATTACHMENT_BIT;
 			tformat.texture_type = RD::TEXTURE_TYPE_2D;
 
-			sky->half_res_pass = RD::get_singleton()->texture_create(tformat, RD::TextureView());
+			skyrd->half_res_pass = RD::get_singleton()->texture_create(tformat, RD::TextureView());
 			Vector<RID> texs;
-			texs.push_back(sky->half_res_pass);
-			sky->half_res_framebuffer = RD::get_singleton()->framebuffer_create(texs);
+			texs.push_back(skyrd->half_res_pass);
+			skyrd->half_res_framebuffer = RD::get_singleton()->framebuffer_create(texs);
 			texture_set_dirty = true;
 		}
 
-		if (sky->quarter_res_pass.is_null() && !RD::get_singleton()->texture_is_valid(sky->quarter_res_pass) && sky->screen_size.x >= 4 && sky->screen_size.y >= 4) {
+		if (skyrd->quarter_res_pass.is_null() && !RD::get_singleton()->texture_is_valid(skyrd->quarter_res_pass) && skyrd->screen_size.x >= 4 && skyrd->screen_size.y >= 4) {
 			RD::TextureFormat tformat;
 			tformat.format = RD::DATA_FORMAT_R16G16B16A16_SFLOAT;
-			tformat.width = sky->screen_size.x / 4;
-			tformat.height = sky->screen_size.y / 4;
+			tformat.width = skyrd->screen_size.x / 4;
+			tformat.height = skyrd->screen_size.y / 4;
 			tformat.usage_bits = RD::TEXTURE_USAGE_SAMPLING_BIT | RD::TEXTURE_USAGE_COLOR_ATTACHMENT_BIT;
 			tformat.texture_type = RD::TEXTURE_TYPE_2D;
 
-			sky->quarter_res_pass = RD::get_singleton()->texture_create(tformat, RD::TextureView());
+			skyrd->quarter_res_pass = RD::get_singleton()->texture_create(tformat, RD::TextureView());
 			Vector<RID> texs;
-			texs.push_back(sky->quarter_res_pass);
-			sky->quarter_res_framebuffer = RD::get_singleton()->framebuffer_create(texs);
+			texs.push_back(skyrd->quarter_res_pass);
+			skyrd->quarter_res_framebuffer = RD::get_singleton()->framebuffer_create(texs);
 			texture_set_dirty = true;
 		}
 
 		if (texture_set_dirty) {
-			for (int i = 0; i < SKY_TEXTURE_SET_MAX; i++) {
-				if (sky->texture_uniform_sets[i].is_valid() && RD::get_singleton()->uniform_set_is_valid(sky->texture_uniform_sets[i])) {
-					RD::get_singleton()->free(sky->texture_uniform_sets[i]);
-					sky->texture_uniform_sets[i] = RID();
+			for (int i = 0; i < RendererSceneSkyRD::RendererSceneSkyRD::SKY_TEXTURE_SET_MAX; i++) {
```

Comment:
This looks like a typo

Reply:
Yes... I do think you are right:) Wonder why the compiler didn't catch that one...